thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/wrapper/ChildKeepingMergeWrapper.java

```diff
@@ -48,9 +50,23 @@ public ChildKeepingMergeWrapper(DataProvider provider, DetectionConfigDTO config
   }
 
   @Override
-  // does not fetch any anomalies from database
```

Comment:
@jihaozh Do you know a reason why we had this logic?

Reply:
Previously, when two detectors detect the same anomaly duration, we retain the parent anomaly's detector name. So next time, when the pipeline runs, the parent anomaly will be retrieved from the rule-level merger.  However, now we changed the parent anomaly's detector name to be the join of two detector names, the rule-level merger won't pick up the parent anomaly. The final merger needs to retrieve those anomalies and merges them.